libraries/AC_WPNav/AC_Circle.cpp

```diff
@@ -107,9 +107,8 @@ void AC_Circle::init(float slew)
     init_start_angle(true);
 
     if(get_rate() > slew) {
-        gcs().send_text(MAV_SEVERITY_CRITICAL, "WARNING: CIRCLE_RATE > ATC_SLEW_YAW");
```

Comment:
I'd prefer we keep the word order as it was because we normally prefix messages with the subsystem as is done here already.

As a side note, I'm not sure the MAV_SEVERITY_CRITICAL is good though. If this is really a warning then we should change this to MAV_SEVERITY_WARNING.